modules/special-rules/pages/super-heavy-weapons.adoc

```diff
@@ -7,48 +8,58 @@ These rules define the effects of super-heavy weapons; see the specific unit dat
 
-== Super-heavy weapons in firefights
-
-* Where a unit that is equipped with super-heavy weapons is in a firefight, and a super-heavy weapon doesn't have a basic Firepower value: Treat that weapon as if it has a Firepower value of 1.
+Overwatch::
+Where a unit that is equipped with super-heavy weapons is on Overwatch orders you may re-roll misses when you shoot with these weapons, just as you do with dice that you get from Firepower.
 
 == Super-heavy weapons and placement of Blast markers
 
 When you shoot with a detachment that includes super-heavy weapons, you may get to place two lots of Blast markers:
 
 * For any regular Firepower that your detachment brings to bear on the target detachment, place Blast markers as normal.
 * For the super-heavy weapons that your detachment brings to bear on the target detachment, place Blast markers as follows.
-+
-|===
-^|Number of super-heavy weapons brought to bear ^|Number of Blast markers to place
-
-^|1 to 3 ^|*
-^|4 to 8 ^|1
-^|9 or more ^|2
-2+|* = 1 Blast marker, but only if you score at least one hit.
-|===
-+
+
+include::main-rules:partial$blast-marker-table.adoc[]
+
 [NOTE]
 ====
 Count the number of super-heavy weapons that you direct at the target detachment.
-Do not modify this number because of Assault orders, Blast markers on the shooting detachment, or anything similar.
 
-+[TODO: Check and consider the wording around this. I seem to remember that there were some inconsistencies or nuances in how Firepower and super-heavy weapons are suppressed by Blast markers. There may be an important difference between how many we "bring to bear" and how many can actually shoot, that also effects Blast marker placement.]+
+* Do *not* modify this number because of Assault orders, Blast markers on the shooting detachment, or anything similar.
+* Do *not* count super-heavy weapons that add to the detachment's Firepower total, such as {barrage} -- the Firepower component already accounts for any Blast markers that they may add.
 ====
 
 == Super-heavy weapons and suppression by Blast markers
 
-With regular Firepower, Blast markers on the detachment each cause the loss of one point of Firepower in the Shooting phase.
-However, where a detachment has only super-heavy weapons available then each Blast marker will normally suppress one entire super-heavy weapon.
-That is, unless the weapon profile on a unit datasheet says otherwise, treat each super-heavy weapon as having a Firepower value of 1.
+Normally, for each Blast marker on a detachment you must reduce its Firepower by 1.
+However, if the detachment has more Blast markers than Firepower, or has no Firepower, then for each remaining Blast marker you lose the ability to shoot with one super-heavy weapon.
+You may choose which super-heavy weapons are suppressed.
+// [TODO: Define suppression properly, perhaps in blast-markers.adoc -- as it is we use the term and imply its meaning, but haven't actually made it explicit.]
```

Comment:
```suggestion
// [TODO: Define suppression properly, perhaps in blast-markers.adoc -- as it is we use the term and imply its meaning, but haven't actually made it explicit.]
```
I think it's OK, as the second bullet explains it as 'you lose the ability to shoot with one super-heavy weapon'.

Reply:
We've have defined it by association here for **SHWs** — but I don't think we've actually defined it clearly for Firepower, in the main rules in spite of the fact that we mention 'suppression' or ' suppressed by' in various places. This is a deficiency in my first pass over the main rules, so I added the TODO note as a prompt to address this.

In `main-rules:blast-markers.adoc`:

>Shooting: For each Blast marker on a detachment, reduce its Firepower by 1. For example, if a detachment with a Firepower of 9 has 3 Blast markers then reduce its Firepower to 6.

The mechanic is there, but no mention of the term suppression, let alone clear definition. I don't think it is in the Shooting phase section either -- in fact, from a quick look, I don't think that the actual mechanic gets a mention in the shooting procedure either, which is another oversight on my part... I might have missed it (or made it a TODO) when I was adding some other notes and analysing text to do with whether suppression should happen at 'S1. Add up shooting Firepower' or at 'https://epicremastered.com/core/main-rules/work-out-how-many-hit-dice-to-roll.html'.

Reply:
I've made some small tweaks to improve 'suppression' references a little. Likely we'll do a bit more with it when we improve the shooting procedure in due course.